src/config_dialog.c

```diff
@@ -113,6 +113,24 @@ static void read_only_cmd(char **words, unsigned int words_len) {
     }
 }
 
+static void find(char **words, unsigned int words_len) {
+    if (words_len == 1) {
+        unsigned int len = strlen(words[0]);
+        int index;
+        for (unsigned int at = 0; at < num_lines && (at != num_lines); ++at) {
+            if (lines[at].length >= len && (index = uchar32_sub(lines[at].data, words[0], lines[at].length, len)) >= 0) {
+                cursor.y = at;
+                cursor.x = index;
+                cursor_in_valid_position();
+                syntaxHighlight();
```

Comment:
Why this call to `syntaxHighlight()`?

Reply:
I added it for error, it is not actually needed

Reply:
Its needed because syntaxHighlight has to be updated every time the lines shown on screen change